tls/_constructs.py

```diff
@@ -19,24 +20,29 @@
     "TLSPlaintext",
     UBInt8("type"),
     ProtocolVersion,
-    UBInt16("length"),  # TODO: Reject packets with length > 2 ** 14
-    Bytes("fragment", lambda ctx: ctx.length),
+    PrefixedBytes("fragment", UBInt16("length")),  # TODO: Reject
+                                                   # packets with
+                                                   # length > 2 ** 14
 )
 
 TLSCompressed = Struct(
     "TLSCompressed",
     UBInt8("type"),
     ProtocolVersion,
-    UBInt16("length"),  # TODO: Reject packets with length > 2 ** 14 + 1024
-    Bytes("fragment", lambda ctx: ctx.length),
+    PrefixedBytes("fragment", UBInt16("length")),  # TODO: Reject
+                                                   # packets with
+                                                   # length > 2 ** 14
+                                                   # + 1024
```

Comment:
Also, how do you feel about putting the comment on its own line (33 here, probably + 24 and 43)